cockatrice/src/gamesmodel.cpp

```diff
@@ -66,15 +65,18 @@ QVariant GamesModel::data(const QModelIndex &index, int role) const
         case ROOM:
             return rooms.value(g.room_id());
         case CREATED: {
-            QDateTime then;
-            then.setTime_t(g.start_time());
-            int secs = then.secsTo(QDateTime::currentDateTime());
-
             switch (role) {
-                case Qt::DisplayRole:
+                case Qt::DisplayRole: {
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 8, 0))
+                    QDateTime then = QDateTime::fromSecsSinceEpoch(g.start_time(), Qt::UTC);
+#else
+                    QDateTime then = QDateTime::fromTime_t(g.start_time(), Qt::UTC);
+#endif
+                    int secs = then.secsTo(QDateTime::currentDateTimeUtc());
                     return getGameCreatedString(secs);
+                }
                 case SORT_ROLE:
-                    return QVariant(secs);
+                    return QVariant(-g.start_time());
```

Comment:
unnecessary performance change :upside_down_face:

Reply:
Not sure what the point of this one is

Reply:
functionally the same, it just avoids calculating the difference with now, consider that this value is only used for sorting

Reply:
actually, I'm not 100% on if this needs an extra cast to not break during 2038 suddenly